src/bo4e/enum/waehrungscode.py

```diff
@@ -0,0 +1,191 @@
+    GMD = "GMD"  #: Dalasi
+    GNF = "GNF"  #: Guinea Franc
+    GTQ = "GTQ"  #: Quetzal
+    GYD = "GYD"  #: Guyana Dollar
+    HKD = "HKD"  #: Hong Kong Dollar
+    HNL = "HNL"  #: Lempira
+    HRK = "HRK"  #: Croatian Kuna
+    HTG = "HTG"  #: Gourde
+    HUF = "HUF"  #: Forint
+    IDR = "IDR"  #: Rupiah
+    ILS = "ILS"  #: New Israeli Sheqel
+    INR = "INR"  #: Indian Rupee
+    IQD = "IQD"  #: Iraqi Dinar
+    IRR = "IRR"  #: Iranian Rial
+    ISK = "ISK"  #: Iceland Krona
+    JMD = "JMD"  #: Jamaican Dollar
+    JOD = "JOD"  #: Jordanian Dinar
+    JPY = "JPY"  #: Yen
+    KES = "KES"  #: Kenyan Shilling
+    KGS = "KGS"  #: Som
+    KHR = "KHR"  #: Riel
+    KMF = "KMF"  #: Comoro Franc
+    KPW = "KPW"  #: North Korean Won
+    KRW = "KRW"  #: Won
+    KWD = "KWD"  #: Kuwaiti Dinar
+    KYD = "KYD"  #: Cayman Islands Dollar
+    KZT = "KZT"  #: Tenge
+    LAK = "LAK"  #: Kip
+    LBP = "LBP"  #: Lebanese Pound
+    LKR = "LKR"  #: Sri Lanka Rupee
+    LRD = "LRD"  #: Liberian Dollar
+    LSL = "LSL"  #: Loti
+    LTL = "LTL"  #: Lithuanian Litas
+    LYD = "LYD"  #: Libyan Dinar
+    MAD = "MAD"  #: Moroccan Dirham
+    MDL = "MDL"  #: Moldovan Leu
+    MGA = "MGA"  #: Malagasy Ariary
+    MKD = "MKD"  #: Denar
+    MMK = "MMK"  #: Kyat
+    MNT = "MNT"  #: Tugrik
+    MOP = "MOP"  #: Pataca
+    MRO = "MRO"  #: Ouguiya
+    MUR = "MUR"  #: Mauritius Rupee
+    MVR = "MVR"  #: Rufiyaa
+    MWK = "MWK"  #: Kwacha
+    MXN = "MXN"  #: Mexican Peso
+    MXV = "MXV"  #: Mexican Unidad de Inversion (UDI)
+    MYR = "MYR"  #: Malaysian Ringgit
+    MZN = "MZN"  #: Mozambique Metical
+    NAD = "NAD"  #: Namibia Dollar
+    NGN = "NGN"  #: Naira
+    NIO = "NIO"  #: Cordoba Oro
+    NOK = "NOK"  #: Norwegian Krone
+    NPR = "NPR"  #: Nepalese Rupee
+    NZD = "NZD"  #: New Zealand Dollar
+    OMR = "OMR"  #: Rial Omani
+    PAB = "PAB"  #: Balboa
+    PEN = "PEN"  #: Nuevo Sol
+    PGK = "PGK"  #: Kina
+    PHP = "PHP"  #: Philippine Peso
+    PKR = "PKR"  #: Pakistan Rupee
+    PLN = "PLN"  #: Zloty
+    PYG = "PYG"  #: Guarani
+    QAR = "QAR"  #: Qatari Rial
+    RON = "RON"  #: New Romanian Leu
+    RSD = "RSD"  #: Serbian Dinar
+    RUB = "RUB"  #: Russian Ruble
+    RUR = "RUR"  #: Russian Ruble
+    RWF = "RWF"  #: Rwanda Franc
+    SAR = "SAR"  #: Saudi Riyal
+    SBD = "SBD"  #: Solomon Islands Dollar
+    SCR = "SCR"  #: Seychelles Rupee
+    SDG = "SDG"  #: Sudanese Pound
+    SEK = "SEK"  #: Swedish Krona
+    SGD = "SGD"  #: Singapore Dollar
+    SHP = "SHP"  #: Saint Helena Pound
+    SLL = "SLL"  #: Leone
+    SOS = "SOS"  #: Somali Shilling
+    SRD = "SRD"  #: Surinam Dollar
+    SSP = "SSP"  #: South Sudanese Pound
+    STD = "STD"  #: Dobra
+    SVC = "SVC"  #: El Salvador Colon
+    SYP = "SYP"  #: Syrian Pound
+    SZL = "SZL"  #: Lilangeni
+    THB = "THB"  #: Baht
+    TJS = "TJS"  #: Somoni
+    TMT = "TMT"  #: Turkmenistan New Manat
+    TND = "TND"  #: Tunisian Dinar
+    TOP = "TOP"  #: PaÊ»anga
```

Comment:
Encoding‽
```suggestion
    TOP = "TOP"  #: Paʻanga
```

Reply:
ich freue mich schon auf unsere erste rechnung in Paʻanga. Wieder ein Fall für #126